frontend/src/components/TableUnratedTaskRow.tsx

```diff
@@ -153,27 +153,22 @@ export const TableUnratedTaskRow: React.FC<TableTaskRowProps> = ({ task }) => {
       <td className={classes(css.unratedTd)}>{numFormat.format(work)}</td>
       <td className={classes(css.unratedTd)}>
         <div className={classes(css.missingContainer)}>
-          {missingRatings && (
-            <div className={classes(css.missingContainer)}>
-              {missingRatings.map((customer) => (
-                <div key={customer.id}>
-                  <Tooltip
-                    classes={{
-                      arrow: classes(css.tooltipArrow),
-                      tooltip: classes(css.tooltip),
-                    }}
-                    title={customer.name}
-                    placement="top"
-                    arrow
-                  >
-                    <div className={classes(css.dotContainer)}>
-                      <Dot fill={customer.color || 'red'} />
-                    </div>
-                  </Tooltip>
-                </div>
-              ))}
-            </div>
-          )}
+          {missingRatings?.map((customer) => (
+            <Tooltip
+              classes={{
+                arrow: classes(css.tooltipArrow),
+                tooltip: classes(css.tooltip),
+              }}
+              key={customer.id}
+              title={customer.name}
+              placement="top"
+              arrow
+            >
+              <div className={classes(css.dotContainer)}>
+                <Dot fill={customer.color || 'red'} />
```

Comment:
Looks like this is a leftover from when customer color could be undefined
```suggestion
                <Dot fill={customer.color} />
```